templates/header.html

```diff
@@ -39,7 +39,9 @@
             <li class="{{if (eq .PageName "ActivistList")}}active{{end}}"><a href="/list_activists">Activists</a></li>
             <li class="{{if (eq .PageName "Leaderboard")}}active{{end}}"><a href="/leaderboard">Leaderboard</a></li>
             <li class="{{if (eq .PageName "Power")}}active{{end}}"><a href="/power">Power</a></li>
+	    <li class="{{if (eq .PageName "UserList")}}active{{end}}"><a href="/admin/users">Users</a></li>
```

Comment:
Is it possible to only show this to admins? ... probably not unless each request has access to the user, which would be a pain in the butt to add unless we have request-local contexts: https://joeshaw.org/revisiting-context-and-http-handler-for-go-17/

Reply:
Thanks for the link. I'll look at the context stuff.